plugin/pkg/scheduler/scheduler.go

```diff
@@ -186,6 +190,13 @@ func (s *Scheduler) scheduleOne() {
 		return
 	}
 
+	// Optimistically assume that the binding will succeed, so we need to invalidate affected
```

Comment:
PS is this `optimistic` or `pessimistic` ?  i think its actually considered pessimistic, since its forcing a recomputation ... right?